Tests/BazelProtobufBindingsTests/BazelProtobufBindingsTests.swift

```diff
@@ -108,4 +107,31 @@ struct BazelProtobufBindingsTests {
 
         #expect(actual == expected)
     }
+
+    @Test
+    func testDecode_streamProto() throws {
```

Comment:
```suggestion
    @Test
    func decodesStreamProto() throws {
```

No more need for the test prefix when using Swift Testing nowadays